apps/frontend/core-dapp/src/features/trade/TradeStore.ts

```diff
@@ -496,9 +496,12 @@ export class TradeStore {
 
     const explorerUrl = hash ? this.root.web3Store.getBlockExplorerUrl(hash) : undefined
     if (error) {
-      toast.error('Trade failed', { description: error, link: explorerUrl })
+      toast.error('Trade Failed', { description: error, link: explorerUrl })
     } else {
-      toast.success('Position opened', { link: explorerUrl })
+      toast.success(
+        tradeMode === 'depositAndTrade' ? 'Deposit & Trade Confirmed' : 'Position Opened',
```

Comment:
We only use `tradeMode` here. Why not just do:

```suggestion
      toast.success(
        this.paymentToken === "native" ? 'Deposit & Trade Confirmed' : 'Position Opened',
```

?

Reply:
yeap good suggestion